Description: Fix _export_hooks called to early to check prototype message.
Bug-Debian: http://bugs.debian.org/320137
Forwarded: no
Author: Don Armstrong <don@debian.org>
Origin: vendor
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-06-21

--- a/Ident.pm
+++ b/Ident.pm
@@ -24,7 +24,7 @@
 );
 
 # provide import magic
-sub _export_hooks () {
+sub _export_hooks {
     my($tag, $hook);
     while ( ($tag, $hook) = each %EXPORT_HOOKS ) {
 	my $hookname = "_export_hook_$tag"; # pseudo-function name
@@ -38,7 +38,7 @@
 # put the export hooks in the standard Exporter structures
 _export_hooks();
 # for compatibility mode, uncomment the next line @@ s/^#\s*// @@
-# @EXPORT = qw(_export_hook_fh);
+@EXPORT = qw(_export_hook_fh);
 
 $VERSION = "1.23";
 
@@ -681,18 +681,18 @@
 # add a paragraph about compatibility mode if appropriate. The non-breaking
 # spaces are to force a new paragraph.
 # @@12 s/^#// @@
-#
-#=pod
-#
-#S< >
-#
-#Adding the B<ident_lookup> method to the B<FileHandle> class used to be
-#automatic in previous version of B<Net::Ident>. During the installation
-#of this B<Net::Ident> package, the system administrator choose to install
-#it in a compatible way, meaning that on this machine, the B<ident_lookup>
-#method is automatically added if you use just C<use Net::Ident;>
-#
-#=cut
+
+=pod
+
+S< >
+
+Adding the B<ident_lookup> method to the B<FileHandle> class used to be
+automatic in previous version of B<Net::Ident>. During the installation
+of this B<Net::Ident> package, the system administrator choose to install
+it in a compatible way, meaning that on this machine, the B<ident_lookup>
+method is automatically added if you use just C<use Net::Ident;>
+
+=cut
 # end of extra paragraph
 
 =pod
